src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/MonitoredEpisodeSpecification.cs

```diff
@@ -34,6 +39,14 @@ public virtual Decision IsSatisfiedBy(RemoteEpisode subject, SearchCriteriaBase
                 return Decision.Reject("Series is not monitored");
             }
 
+            // If indexer has tags, check that at least one of them is present on the series
+            var indexerTags = _indexerRepository.Get(subject.Release.IndexerId).Tags;
+            if (!indexerTags.Empty() && indexerTags.Intersect(subject.Series.Tags).Empty())
```

Comment:
Nit: new line before if block.

Reply:
`indexerTags.Any()` instead of the double negative.

Reply:
These changes need their own specification, not part of a different specification (as well as their own file for the tests).